planetarium/NineChronicles
# Design/balance v2

### Description

1. Fix Equipment item's life cycle

### How to test

1. Check stage 1~200 item drop icons.
2. Check recipe's stage unlock requirements.

### Related Links

### Required Reviewers

@Namyujeong 

### Screenshot
![image](https://user-images.githubusercontent.com/60082897/132838095-e44ba610-0021-4033-b483-0acba469e2e4.png)
![image](https://user-images.githubusercontent.com/60082897/132838588-72f5e917-707e-44f3-b7a7-cc263ef9b993.png)
